operators/src/adapters/raster_subquery_adapter.rs

```diff
@@ -611,6 +611,17 @@ where
         let output_raster =
             EmptyGrid::new(tile_info.tile_size_in_pixels, self.no_data_and_fill_value);
 
+        let proj = CoordinateProjector::from_known_srs(self.out_srs, self.in_srs)?;
+        let valid_bounds_proj =
+            CoordinateProjector::from_known_srs(SpatialReference::epsg_4326(), self.out_srs)?;
+
+        let valid_bounds = self
+            .in_srs
+            .area_of_use::<SpatialPartition2D>()?
+            .reproject(&valid_bounds_proj)?;
```

Comment:
I'm not 100% sure why 4326 is used every time.
I guess this could be a separate function and have a documentation of what this is.

Reply:
the is a `area_of_use_projected` method which you can use instead